pytorch_lightning/plugins/checkpoint/checkpoint.py

```diff
@@ -0,0 +1,58 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
Rename  the file to `checkpoint_io_plugin.py` to fit the class?

or even better

`pytorch_lightning/plugins/io/checkpoint_plugin.py`